ZenPacks/zenoss/Import4/service_definition/imp4mariadb.json

```diff
@@ -195,7 +195,7 @@
        "Actions": null,
        "HealthChecks": {
          "running": {
-           "Script": "which rrdtool && mysql --socket=/var/lib/mysql/mysql.sock -uroot -e 'select 1' && mysql --socket=/var/lib/mysql.model/mysql.sock -uroot -e 'select 1' > /dev/null",
+           "Script": "which rrdtool && [ 2 -eq $(ps -ef | grep '/bin/bash /import4/pkg/bin/imp4mariadb.sh' | wc -l) ] > /dev/null",
```

Comment:
Yes, pgrep works.. Code updated and commited.
Thanks!

On Wed, May 13, 2015 at 12:26 PM, Phil Lu <notifications@github.com> wrote:

> In ZenPacks/zenoss/Import4/service_definition/imp4mariadb.json
> <https://github.com/zenoss/ZenPacks.zenoss.Import4/pull/30#discussion_r30253897>
> :
>
> > @@ -195,7 +195,7 @@
> >         "Actions": null,
> >         "HealthChecks": {
> >           "running": {
> > -           "Script": "which rrdtool && mysql --socket=/var/lib/mysql/mysql.sock -uroot -e 'select 1' && mysql --socket=/var/lib/mysql.model/mysql.sock -uroot -e 'select 1' > /dev/null",
> > +           "Script": "which rrdtool && [ 2 -eq $(ps -ef | grep '/bin/bash /import4/pkg/bin/imp4mariadb.sh' | wc -l) ] > /dev/null",
>
> pgrep -fla '/bin/bash /import4/pkg/bin/imp4mariadb.sh' | wc -l
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zenoss/ZenPacks.zenoss.Import4/pull/30/files#r30253897>
> .
>



-- 
Thanks,
----
Alex